dist/gaia-header.js

```diff
@@ -913,11 +918,40 @@ module.exports = component.register('gaia-header', {
   observerStop: function() {
     if (!this.observing) { return; }
     this.observer.disconnect();
+    window.removeEventListener('resize', this);
+    this._resizeHandled = false;
+
     this.observing = false;
     debug('observer stopped');
   },
 
   /**
+   * Handle DOM events
+   */
+  handleEvent: function(e) {
```

Comment:
I've never really been a fan of this pattern. But willing to be convinced otherwise :)